src/pylorax/creator.py

```diff
@@ -49,7 +49,7 @@
 
 # Default parameters for rebuilding initramfs, override with --dracut-arg or --dracut-conf
 DRACUT_DEFAULT = ["--xz", "--add", "livenet dmsquash-live dmsquash-live-ntfs convertfs pollcdrom qemu qemu-net",
```

Comment:
include `dmsquash-live-autooverlay` to address https://bugzilla.redhat.com/show_bug.cgi?id=2170544

Reply:
Yeah, that should be a separate set of patches since it's not directly related.